clients/src/main/java/org/apache/kafka/common/requests/GetTelemetrySubscriptionRequest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.common.requests;
+
+import org.apache.kafka.common.Uuid;
+import org.apache.kafka.common.message.GetTelemetrySubscriptionRequestData;
+import org.apache.kafka.common.message.GetTelemetrySubscriptionResponseData;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.ByteBufferAccessor;
+import org.apache.kafka.common.protocol.Errors;
+
+import java.nio.ByteBuffer;
+
+public class GetTelemetrySubscriptionRequest extends AbstractRequest {
+
+    public static class Builder extends AbstractRequest.Builder<GetTelemetrySubscriptionRequest> {
+        Uuid clientInstanceId;
```

Comment:
Are these instance variables traditionally package private, or should they have builder-pattern methods? I don't care and I assume this is how the other requests are too, but I just wanted to be sure.

Reply:
I am not sure either, I have just followed the existing convention by mimicking one of the existing methods